app/models/offer.rb

```diff
@@ -41,6 +41,10 @@ def price=(value)
   end
 
   def total
-    price * (1 + billing_profile.vat_rate)
+    if billing_profile.present?
+      price * (1 + billing_profile.vat_rate)
```

Comment:
Maybe extract method can help to better explain this magic number?

Reply:
Or just a freezed named constant value?